tests/security_tests.rs

```diff
@@ -0,0 +1,305 @@
+use axum_gate::jsonwebtoken::{DecodingKey, EncodingKey, Header, Validation};
+use axum_gate::jwt::{JsonWebToken, JsonWebTokenOptions, JwtClaims, RegisteredClaims};
+use axum_gate::hashing::{Argon2Hasher, VerificationResult};
+use axum_gate::secrets::Secret;
+use axum_gate::services::{CodecService, HashingService};
+use axum_gate::{Account, Credentials, Group, Role};
+use chrono::{TimeDelta, Utc};
```

Comment:
Imports of the library, std and externals should be done in three different paragraphs.